client/src/components/Public/Map/AddressDetail/ListRow.js

```diff
@@ -17,6 +17,12 @@ import {entityDetailSelector} from '../../../../selectors'
 import Info from '../../../shared/Info/Info'
 import './ListRow.css'
 
+
+type DetailedInfoProps = {
+  toggleEntityInfo: (id: number) => void,
+  data: Object,
```

Comment:
Data is typed in `Info` component, please, use that one insted of general `Object`.